.changeset/tender-rabbits-speak.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/datastore": minor
+---
+---
+"section": deprecation
+---
+
+The function `process` on `FluidDataStoreRuntime` is now deprecated.
+
+A new function `processMessages` has been added in its place which will be called to process multiple messages instead of a single one on the data store runtime. This is part of a feature called "Op bunching" where contiguous ops of a given type and to a given data store / DDS are bunched and sent together for processing.
```

Comment:
While both exist back-compat, I assume we're being smart enough when we call them such that a given message doesn't get processed by both. Is there anything related to that that consumers need to know?

Reply:
Added a note mentioning that process may still be called in compat scenarios.